homework-02-relaxr-landing-page-master/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="Landing page">
+    	<meta name="keywords" content="one, two, three">
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+
+		<title>Relaxr</title>
+
+		<!-- external CSS link -->
+		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/style.css">
+
+	</head>
+	<body>
+		<div id="container">
+			<header>
+				<h1>Relaxr</h1>
+				<h2>Get piece of mind with a single tap</h2>
+						<div class="getbutton">
+							<p>	Get it Now</p>
+						</div>
+			</header>
+				<div class="benefits">
+					<h1>Benefits</h1>
+			 			<p>The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.
+			 			</p>
+			 			<ul>
+			 				<li>Schedule meetings for you</li>
+			 				<li>Excel automation to complete your work for you</li>
+			 				<li>Resonds to emails on your behalf</li>
+			 				<li>Does all your work for you with our revolutionary AI technology</li>
+						</ul>
+			 	</div>
+			 	<div class="test">	
+					<div class="quote">
+						<p>	Relaxr changed my life. I've been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work</p>
+					</div>
+					<p class="name"> - Amanda, Intuit</p>
+					
+					<div class="yellowsec">	
+						<div class="sign">
+							<p>Sign Up Now</p>
```

Comment:
@cliffjadoo the `p` tag should only be used for text. To make buttons either use the `button` tag or an `a` tag and style it accordingly 🔨